mmdet/apis/export.py

```diff
@@ -124,6 +124,15 @@ def add_node_names(export_name):
     onnx.save(model, export_name)
 
 
+def _get_mo_cmd():
+    for mo_cmd in ('mo', 'mo.py'):
```

Comment:
Under what circumstances we end up lacking `mo` command, if we always install it via pip?

Reply:
Initially I used the code with OpenVINO 2021.4 installed from a package, so this code was required.
Later I switched to `init_venv.sh` and used OpenVINO pip package.

But note that sometimes we still **need** using OpenVINO package. As an example,  if we need to check OpenVINO latest features -- this week I had to get recent OpenVINO package from OpenVINO CI as a set of zip files, extract them, and use `setupvars.sh`
(and yes, after that Model Optimizer is `mo.py` instead of `mo`)

So, I propose to keep this change, since it may be useful.

Reply:
Looks more like a legacy support effort, but Ok, let's leave it as is.